### PR TITLE
2nd difference for wtm auto switch

### PR DESCRIPTION
**Pull request by papampi**
v0018: code clean 
v0019: Lower difference when minimum mining time reached

### CODE REVIEW

**Comment by papampi** [comment]
@LuKePicci 
Possible to make WTM_SWITCHER read config from command line?

**Comment by LuKePicci** [comment]
Sure, what params exactly? all I see in the json file?

**Comment by papampi** [comment]
Yes, instead of variable point it to command line
I was thinking it reads from command line and when I test I noticed it reads from its variable

**Comment by papampi** [comment]
So in 8wtm_auto_switch the old line works:
    python2.7 '/home/m1/WTM_SWITCHER' /home/m1/WTM.json

**Comment by LuKePicci** [comment]
take a look here http://www.diveintopython.net/scripts_and_streams/command_line_arguments.html

File: 3main [hunk 1]
```diff
@@ -1057,6 +1060,13 @@ then
  "WTM_MIN_DIFFERENCE": "$WTM_MIN_DIFFERENCE",
  "currency":"$WTM_CURRENCY"
 }
+EOF
+{
```
Comment (LuKePicci) [inline 3main]: There is something strange to me. Was it intended to replace the previous block or to be an alternative? Does this syntax works?

**Comment by papampi** [comment]
Thanks mate, I have been reading and googling it for past 2 days, but couldnt make it work

**Comment by LuKePicci** [comment]
If I'm not wrong, salfter switch works somehow the same, as I can see from 3main the json config file is specified by command line.

**Comment by papampi** [comment]
Got it, Thanks
I was trying to change the configFile variable
changed cfg to 
cfg=json.loads(open(sys.argv[1]).read())
and all is good

**Review by LuKePicci — APPROVED** [review]

File: 3main (hunk 1 above)
Comment (papampi) [reply 3main]: Yup, had a missing line, fixed it.

**Pull request merged**